lib/codegen/meta/isa/x86/recipes.py

```diff
@@ -544,6 +545,24 @@ def floatccs(iform):
         sink.put8(imm as u64);
         ''')
 
+# XX /n Unary with floating point 32-bit immediate equal to zero.
+f32imm_z = TailRecipe(
+    'f32imm_z', UnaryIeee32, size=4, ins=(), outs=FPR,
```

Comment:
The declared size will need to be updated too. Note that this is the size of the "tail" portion of the encoding, which is just the one modrm byte.